mappings/net/minecraft/block/NoteBlock.mapping

```diff
@@ -2,9 +2,11 @@ CLASS net/minecraft/unmapped/C_nzuwmjdt net/minecraft/block/NoteBlock
 	FIELD f_ajsbzucp NOTE Lnet/minecraft/unmapped/C_vltzvhxi;
 	FIELD f_aucyqdcc INSTRUMENT Lnet/minecraft/unmapped/C_cgckxfsw;
 	FIELD f_kmjcamoe POWERED Lnet/minecraft/unmapped/C_xhwijdsd;
+	FIELD f_zcrvmfco NOTE_VOLUME I
 	METHOD m_eelivhoc getStateWithInstrument (Lnet/minecraft/unmapped/C_vdvbsyle;Lnet/minecraft/unmapped/C_hynzadkk;Lnet/minecraft/unmapped/C_txtbiemp;)Lnet/minecraft/unmapped/C_txtbiemp;
 		ARG 1 worldAccess
-	METHOD m_umbeungg mobHeadSoundsEnabled (Lnet/minecraft/unmapped/C_vdvbsyle;)Z
+	METHOD m_iwpuznep getPlayerHeadNoteSound (Lnet/minecraft/unmapped/C_cdctfzbn;Lnet/minecraft/unmapped/C_hynzadkk;)Lnet/minecraft/unmapped/C_ncpywfca;
```

Comment:
could be made more concise, I think the `Note` is fairly redundant here

Reply:
Ah, I was worried that it would be too concise to the point where it loses a bit of meaning; Maybe I could either remove `Note` or `Sound`...? maybe the former

Reply:
personal preference would be note, but either removal works!